src/style/ngm-object-information.css

```diff
@@ -2,6 +2,7 @@ ngm-object-information {
   position: absolute;
   z-index: 2;
   width: 528px;
+  max-width: 100%;
   max-height: 643px;
```

Comment:
need to adapt height too

![image](https://user-images.githubusercontent.com/51954170/152522683-979f89b1-66e3-4d2c-b9b6-aedd9dc3e83e.png)

Reply:
fixed, can you check again ?